packages/jest/src/generators/jest-project/lib/create-files.ts

```diff
@@ -13,10 +13,13 @@ export function createFiles(tree: Tree, options: JestProjectSchema) {
   const filesFolder =
     options.setupFile === 'angular' ? '../files-angular' : '../files';
 
+  const supportTsJestConfig =
+    options.transformer === 'ts-jest' || options.transformer === 'babel-jest';
+
```

Comment:
wouldn't we only support the TsJest config if the transformer is ts-jest? I could have missed something here, so feel free to correct me.